### ISSUE TITLE
MSI install fails, admin issues?

### ISSUE THREAD

**Issue by jdx-john** [prometheus-community/windows_exporter#889]
I've got the latest amd64 MSI, and from a normal CMD window ran:
`msiexec /i <path-to-msi-file> LISTEN_PORT=5000`

I got a popup:
![image](https://user-images.githubusercontent.com/1654627/146574561-3694bb9c-5204-4621-b523-026696de965e.png)

So I tried from CMD in administrator mode, same result.
I wondered if maybe that `/i` option is wrong and tried with `/a` but then I got the following:
![image](https://user-images.githubusercontent.com/1654627/146574721-c736464a-13a2-4ec5-81ae-95a1599b7419.png)

I would expect it needs admin rights to install a service but not sure how to proceed. This _is_ a legacy Windows7 machine - is it possible WE is non-functional on this old OS?

**Comment by breed808** [comment]
The `/i` flag is correct, that's what we've been using to install the exporter for quite some time now. Windows 7 shouldn't be an issue either, it's currently supported by the exporter.

Is the `<path-to-msi-file>` an absolute or relative path? I've noticed some errors when using relative paths for the MSI file in the past.

**Comment by jdx-john** [comment]
I think I can close this one off, happily. I raised with the sysadmin and it turns out the user I logged in as was much more restricted than I had thought. So even when elevating to administrator, it didn't have the correct rights.

Googling error 2203 threw up results about permissions copying files to restricted folders which is what led me to ask IT. Using a different login I'm happy to report I have it working smoothly :)
